tox.ini

```diff
@@ -148,4 +148,5 @@ commands=
   update_cluster: {[update]commands}
   update_docker_cluster: {[update]commands}
 
+  sleep 3h
```

Comment:
I haven't looked at a log, but that's probably because ``sleep`` isn't whitelisted here https://github.com/ceph/ceph-ansible/blob/master/tox.ini#L56